diff --git a/uefi-sct/SctPkg/CommonGenFramework.sh b/uefi-sct/SctPkg/CommonGenFramework.sh
index e7b7ac72..70083fd9 100755
--- a/uefi-sct/SctPkg/CommonGenFramework.sh
+++ b/uefi-sct/SctPkg/CommonGenFramework.sh
@@ -379,6 +379,15 @@ then
     cp $ProcessorType/ImageServicesBBTest.efi                  $Framework/Test/ > NUL
     cp $ProcessorType/MiscBootServicesBBTest.efi               $Framework/Test/ > NUL
 
+    # BBSR
+    cp $ProcessorType/BBSRVariableSizeTest.efi                 $Framework/Test/ > NUL
+    cp $ProcessorType/TCGMemoryOverwriteRequestBBTest.efi      $Framework/Test/ > NUL
+    cp $ProcessorType/TCG2ProtocolBBTest.efi                   $Framework/Test/ > NUL
+  if [ $SECUREBOOT_ENABLE -eq 1 ]; then
+    cp $ProcessorType/SecureBootBBTest.efi                     $Framework/Test/ > NUL
+  fi
+    cp $ProcessorType/PlatformResetAttackMitigationPsciTest.efi $Framework/Test/ > NUL
+
     cp $ProcessorType/VariableServicesBBTest.efi               $Framework/Test/ > NUL
     cp $ProcessorType/TimeServicesBBTest.efi                   $Framework/Test/ > NUL
     cp $ProcessorType/MiscRuntimeServicesBBTest.efi            $Framework/Test/ > NUL
@@ -534,4 +543,7 @@ then
     CopyDependency PciRootBridgeIo
     CopyDependency PxeBaseCode
     CopyDependency ConfigKeywordHandler
-fi
\ No newline at end of file
+  if [ $SECUREBOOT_ENABLE -eq 1 ]; then
+    CopyDependency SecureBoot
+  fi
+fi
diff --git a/uefi-sct/SctPkg/Config/Data/Category.ini b/uefi-sct/SctPkg/Config/Data/Category.ini
index 1e88a593..5237dd47 100644
--- a/uefi-sct/SctPkg/Config/Data/Category.ini
+++ b/uefi-sct/SctPkg/Config/Data/Category.ini
@@ -172,6 +172,13 @@ InterfaceGuid = AFF115FB-387B-4C18-8C41-6AFC7F03BB90
 Name          = RuntimeServicesTest\VariableServicesTest
 Description   =
 
+[Category Data]
+Revision      = 0x00010000
+CategoryGuid  = d250c2ca-0fe7-4edf-9319-6eef4f6b0d12
+InterfaceGuid = AFF115FB-387B-4C18-8C41-6AFC7F03BB90
+Name          = RuntimeServicesTest\BBSRVariableSizeTest
+Description   =
+
 [Category Data]
 Revision      = 0x00010000
 CategoryGuid  = 0C1D662C-EA60-48F5-9D77-22F853499C4A
@@ -193,13 +200,13 @@ InterfaceGuid = AFF115FB-387B-4C18-8C41-6AFC7F03BB90
 Name          = RuntimeServicesTest\MiscRuntimeServicesTest
 Description   =
 
-[Category Data]
-Revision      = 0x00010000
-CategoryGuid  = 265E661D-D51E-454B-BBC9-665F46EC0A04
-InterfaceGuid = AFF115FB-387B-4C18-8C41-6AFC7F03BB90
-Name          = RuntimeServicesTest\TCGMemoryOverwriteRequestTest
-Description   = Test TCG Platform Reset Attack Mitigation EFI variables
-
+[Category Data]
+Revision      = 0x00010000
+CategoryGuid  = 265E661D-D51E-454B-BBC9-665F46EC0A04
+InterfaceGuid = AFF115FB-387B-4C18-8C41-6AFC7F03BB90
+Name          = RuntimeServicesTest\TCGMemoryOverwriteRequestTest
+Description   = Test TCG Platform Reset Attack Mitigation EFI variables
+
 [Category Data]
 Revision      = 0x00010000
 CategoryGuid  = 52A69BBF-E19F-4e82-93D6-AAC46E2E1E06
@@ -1074,10 +1081,17 @@ CategoryGuid  = CCD15FEC-6F73-4EEC-8395-3E69E4B940BF
 InterfaceGuid = CCD15FEC-6F73-4EEC-8395-3E69E4B940BF
 Name          = RiscvBootProtocolTest
 Description   = RISC-V Boot Protocol Test
-
-[Category Data]
-Revision      = 0x00010000
-CategoryGuid  = 607f766c-7455-42be-930b-e4d76db2720f
-InterfaceGuid = 607f766c-7455-42be-930b-e4d76db2720f
-Name          = TCG2ProtocolTest
-Description   = TCG2 Protocol Test
+
+[Category Data]
+Revision      = 0x00010000
+CategoryGuid  = 607f766c-7455-42be-930b-e4d76db2720f
+InterfaceGuid = 607f766c-7455-42be-930b-e4d76db2720f
+Name          = TCG2ProtocolTest
+Description   = TCG2 Protocol Test
+
+[Category Data]
+Revision      = 0x00010000
+CategoryGuid  = C4567D24-BB60-484E-80EC-A36ACEEA5C43
+InterfaceGuid = 71652D04-BF38-434A-BCB8-6547D7FD8384
+Name          = GenericTest\PlatformResetAttackMitigationPsciTest
+Description   = Check Platform Reset Attack Mitigation Via PSCI
