diff --git a/uefi-sct/SctPkg/CommonGenFramework.sh b/uefi-sct/SctPkg/CommonGenFramework.sh
index c9ba7d04..a59b2945 100755
--- a/uefi-sct/SctPkg/CommonGenFramework.sh
+++ b/uefi-sct/SctPkg/CommonGenFramework.sh
@@ -411,6 +411,15 @@ then
     cp $ProcessorType/ImageServicesBBTest.efi                  $Framework/Test/ > NUL
     cp $ProcessorType/MiscBootServicesBBTest.efi               $Framework/Test/ > NUL
 
+    # BBSR
+    cp $ProcessorType/BBSRVariableSizeTest.efi                 $Framework/Test/ > NUL
+    cp $ProcessorType/TCGMemoryOverwriteRequestBBTest.efi      $Framework/Test/ > NUL
+    cp $ProcessorType/TCG2ProtocolBBTest.efi                   $Framework/Test/ > NUL
+  if [ $SECUREBOOT_ENABLE -eq 1 ]; then
+    cp $ProcessorType/SecureBootBBTest.efi                     $Framework/Test/ > NUL
+  fi
+    cp $ProcessorType/PlatformResetAttackMitigationPsciTest.efi $Framework/Test/ > NUL
+
     cp $ProcessorType/VariableServicesBBTest.efi               $Framework/Test/ > NUL
     cp $ProcessorType/TimeServicesBBTest.efi                   $Framework/Test/ > NUL
     cp $ProcessorType/MiscRuntimeServicesBBTest.efi            $Framework/Test/ > NUL
@@ -569,4 +578,7 @@ then
     CopyDependency PciRootBridgeIo
     CopyDependency PxeBaseCode
     CopyDependency ConfigKeywordHandler
+  if [ $SECUREBOOT_ENABLE -eq 1 ]; then
+    CopyDependency SecureBoot
+  fi
 fi
diff --git a/uefi-sct/SctPkg/Config/Data/Category.ini b/uefi-sct/SctPkg/Config/Data/Category.ini
index fbe7659f..070d73ef 100644
--- a/uefi-sct/SctPkg/Config/Data/Category.ini
+++ b/uefi-sct/SctPkg/Config/Data/Category.ini
@@ -1081,3 +1081,17 @@ CategoryGuid  = 14BC21EC-6D04-4684-A892-1B465ACAC423
 InterfaceGuid = 71652D04-BF38-434A-BCB8-6547D7FD8384
 Name          = GenericTest\EfiConformanceProfileTableTest
 Description   = Efi Conformance Profile Table Test
+
+[Category Data]
+Revision      = 0x00010000
+CategoryGuid  = d250c2ca-0fe7-4edf-9319-6eef4f6b0d12
+InterfaceGuid = AFF115FB-387B-4C18-8C41-6AFC7F03BB90
+Name          = RuntimeServicesTest\BBSRVariableSizeTest
+Description   =
+
+[Category Data]
+Revision      = 0x00010000
+CategoryGuid  = C4567D24-BB60-484E-80EC-A36ACEEA5C43
+InterfaceGuid = 71652D04-BF38-434A-BCB8-6547D7FD8384
+Name          = GenericTest\PlatformResetAttackMitigationPsciTest
+Description   = Check Platform Reset Attack Mitigation Via PSCI
